lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -475,19 +545,26 @@ static lldb::ThreadPlanSP GetStepThroughTrampolinePlan(Thread &thread,
       log->Printf(
           "Stepped to thunk \"%s\" (kind: %s) stepping to target: \"%s\".",
           symbol_name, GetThunkKindName(thunk_kind), thunk_target.c_str());
+    return CreateRunToAddressPlan(thunk_target, thread, stop_others);
+  }
+  case ThunkAction::RunToObjcCInteropCtor: {
+    LLDB_LOG(log, "SwiftLanguageRuntime: running to "
```

Comment:
Can you wait the log output till you've got the class name?  It seems useful to know what class we thought we were supposed to run to.

Reply:
You could also then log the case where somebody passes you a demangled name you couldn't find the class it should have targeted, which would also be nice to see.